### PR TITLE
feat(verify): add gofmt govet boilerplate test

### PR DESCRIPTION
**Pull request by zjj2wry**
**What this PR does**
1. add gofmt govet boilerplate test
2. execute hack/update-all.sh

**why we need it**

**Special notes for your reviewer**:
/cc @ddysher

### CODE REVIEW

**Comment by Jimexist** [comment]
govet 可以加到 gometalinter 上面

**Comment by zjj2wry** [comment]
好的， go fmt 已经有了吗？

**Comment by ddysher** [comment]
@zjj2wry @Jimexist Any updates?

**Comment by zjj2wry** [comment]
看配置似乎不能配置多个语言～

**Comment by zjj2wry** [comment]
@ddysher @Jimexist gofmt 和 govet 已经加到 ci 里面了

File: hack/boilerplate/boilerplate.py.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Copyright YEAR Caicloud Authors
```
Comment (ddysher) [inline hack/boilerplate/boilerplate.py.txt]: we already have this under root directory.

**Comment by ddysher** [comment]
@zjj2wry why not just add `gofmt` and `go vet` to gometalinter?

**Comment by zjj2wry** [comment]
@ddysher ``` - gometalinter --config=config.json ./...``` will check all package. and i want add `gofmt -s -d` and -r=rules, it is not easy extensional

File: hack/boilerplate/boilerplate.py.txt (hunk 1 above)
Comment (zjj2wry) [reply hack/boilerplate/boilerplate.py.txt]: i move it into `hack` package. and add python and sh boilderplate

File: hack/update-all.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+TESTINFRA_ROOT=$(git rev-parse --show-toplevel)
+${TESTINFRA_ROOT}/hack/update-gofmt.sh
+go run ./cmd/license/apache.go --go-header-file hack/boilerplate/boilerplate.go.txt --logtostderr
```
Comment (zoumo) [inline hack/update-all.sh]: need newline

File: hack/verify-gofmt.sh [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+set -o nounset
+set -o pipefail
+
+diff=$(find . -name "*.go" | grep -v "\/vendor\/" | xargs gofmt -s -d 2>&1)
+if [[ -n "${diff}" ]]; then
+  echo "${diff}"
+  echo
+  echo "Please run hack/update-gofmt.sh"
+  exit 
+fi
```
Comment (zoumo) [inline hack/verify-gofmt.sh]: newline

File: Makefile [hunk 4]
```diff
@@ -18,5 +21,5 @@ flag-gen:
 
 .PHONY: license
 license: 
-	go run ./cmd/license/apache.go --go-header-file boilerplate/boilerplate.go.txt --logtostderr
-
+	go run ./cmd/license/apache.go --go-header-file hack/boilerplate/boilerplate.go.txt --logtostderr
+
```
Comment (zoumo) [inline Makefile]: newline

**Comment by zjj2wry** [comment]
@zoumo address, ptal

File: hack/verify-gofmt.sh (hunk 3 above)
Comment (zjj2wry) [reply hack/verify-gofmt.sh]: done

File: hack/update-all.sh (hunk 2 above)
Comment (zjj2wry) [reply hack/update-all.sh]: done

**Comment by zjj2wry** [comment]
@zoumo @ddysher ptal

**Comment by ddysher** [comment]
LGTM

@zjj2wry Can you add a README in `hack`?

**Comment by ddysher** [comment]
In a new PR

**Pull request merged**